Kajabi/sage-lib
# SPIKE: a11y-friendly disabled state

## Description
Testing an option for more inclusive `disabled` controls. Should help allow secondary actions (tooltips, etc) but still prevent form submissions/actions unless requirements are met.

References:
- https://css-tricks.com/making-disabled-buttons-more-inclusive/
- https://codepen.io/stevef/post/short-note-on-the-disabled-attribute

## Testing in `sage-lib`
1. Navigate to the "button" component in the sage docs
2. The last button in each example row has a tooltip enabled. Hover your mouse over the active (non-disabled) example to confirm a tooltip appears, and the button can be clicked/toggled via keypress
3. Hover over the "disabled" (`aria-disabled`) tooltip example, and confirm a tooltip appears but the button **cannot** be actuated


## Testing in `kajabi-products`

1. (**MEDIUM**) Disabled button states use a combination of JS and `aria-disabled` to emulate the `disabled` attribute
